modules/sdk-api/src/types.ts

```diff
@@ -88,6 +88,7 @@ export interface AuthenticateOptions {
   forceSMS?: boolean;
   extensible?: boolean;
   forceV1Auth?: boolean;
+  ensureEcdhKeychain?: boolean;
```

Comment:
I know the other fields dont have a description, but it would be good to have a small comment here on what this field represents? What happens if its true/false, etc.